installers/npm/binary.js

```diff
@@ -47,6 +48,13 @@ const getPlatform = () => {
       type === supportedPlatform.TYPE &&
       architecture === supportedPlatform.ARCHITECTURE
     ) {
+      if (supportedPlatform.TYPE === "Linux") {
+        if (!existsSync("/lib/x86_64-linux-gnu/libc.so.6")) {
```

Comment:
Is this that predictable across platforms?  I surprisingly haven't thought about this much ever. 
 Anything you can offer to appease my curiosity?

I'm mostly just curious if it's defensive enough or durable enough.  A person who is me might have suggested just invoking `ldd --version` (e.g., `execSync`) and if that errors, then presume we're unable to proceed.

Reply:
ldd exists and uses musl on Alpine so that approach wouldn't quite work. I'm not 100% sure this is durable tbh, I just know it exists on Ubuntu and not Alpine and... its what rust tries to link to for the gnu Target

Reply:
@EverlastingBugstopper 

This is on a CentOS.  Probably also on the `centos` Docker image, if you wanted to test:

```
$ ls -la /lib/x86_64-linux-gnu/libc.so.6
ls: cannot access '/lib/x86_64-linux-gnu/libc.so.6': No such file or directory
$ uname -a
Linux www-252 4.15.0-142-generic #146-Ubuntu SMP Tue Apr 13 01:11:19 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux
```

Reply:
I should add:

```
$ ldd --version
ldd (GNU libc) 2.28
Copyright (C) 2018 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
Written by Roland McGrath and Ulrich Drepper.
```

Reply:
mm yeah, seems like this is not the correct approach then